Extensions/classic_header.js

```diff
@@ -92,6 +112,17 @@ XKit.extensions.classic_header = new Object({
 			XKit.tools.add_css(" .tab_notice_value { color: #ffffff !important; } .selected .tab_notice, .tab_notice { background: #bc3333 !important; } .tab_bar .tab.selected .tab_anchor, .tab_bar .tab.active .tab_anchor {opacity: 0.5;}", "classic_header_fixed_color");
 		}
 
+		if (XKit.extensions.classic_header.preferences.mobile_sticky.value === true) {
```

Comment:
Were we not doing away with the `=== true/false` checks and just leaving it as `value`?